wpiformat/wpiformat/__init__.py

```diff
@@ -227,6 +227,11 @@ def main():
         help=
         "verbosity level 2 (prints names of processed files and tasks run on them)"
     )
+    parser.add_argument(
+        "-list-files",
+        dest="list_files",
+        action="store_true",
+        help="list files to be processed instead of processing them")
```

Comment:
This description is incorrect because it still processes the files after listing them.

Reply:
Fixed